requirements.yml

```diff
@@ -190,9 +190,9 @@
   setup_options: '--offline'
   include_extras: false
 
-#- name: glueviz
-#  version: '0.9.1'
-#  python: '<3.6*'
+- name: glueviz
+  version: '0.10.0'
+  python: '<3.6*'
```

Comment:
You're right. Though I think that version limitation in ``requirements.yml`` is ignored at the moment (thus the reproject failure).